src/index.hot.js

```diff
@@ -1,19 +1,20 @@
+/* eslint global-require: 0 */
+import 'babel-polyfill';
```

Comment:
I ran `` npm start -- --config webpack.prod.config.js --open `` at 606fc75 and I can succ
![screenshot-localhost 8080-2016-12-18-22-52-55](https://cloud.githubusercontent.com/assets/3063067/21294310/d3fc3370-c574-11e6-9fce-50569921dec1.png)